### ISSUE TITLE
A SyntaxError occurs during JSON parsing if the input parameter hasn't been stringified.

### ISSUE THREAD

**Issue by lenonMax** [digitalfortress-tech/localstorage-slim#25]
### What happened?

When a key's value pre-exists as a string or has been previously set using the localStorage.setItem() method in local storage, a JSON parsing error arises upon invoking the ls.get() method.

### Affected version

Latest

### Reproduction URL

https://jsfiddle.net/j3qhtv28/5/

### How to reproduce?

1. Go to the provided reproduction link
2. Click on 'Console'
3. See the error

### Browsers

Chrome

### Relevant log output

```Shell
Uncaught SyntaxError: Unexpected token 'd', "d" is not valid JSON
    at JSON.parse (<anonymous>)
    at <anonymous>:1:6
```

### Additional context

I observed that this issue was addressed in the [commit](https://github.com/digitalfortress-tech/localstorage-slim/pull/22/commits/7ac88990bdfc24f65b4a6047e2c820bc8bfe6627). Do you plan to merge it into the main branch soon? If not, I'm willing to submit another request to expedite the fix, as it's crucial to resolve this promptly. This is a common scenario many front-end developers might encounter when launching a new app on the default port, especially if there's a residual "token" value in the local storage not set by the ls Object. Addressing this will prevent potential confusion.

**Comment by niketpathak** [comment]
Hello @lenonMax , thank you for taking the time to report the issue. It has been fixed in [v2.5.0](https://www.npmjs.com/package/localstorage-slim).

**Comment by lenonMax** [comment]
@niketpathak Thank you for your prompt response and for quickly addressing the issue. I appreciate your dedication to this project and am glad to see it continuously improving. The work you and the team are doing is terrific!
